vision_project/vision_models/imageloader.py

```diff
@@ -137,21 +158,53 @@ def _combine_labels(self, labels):
         """
         return tf.reduce_sum(labels, axis=1)
 
+    def split_dataset(self):
+        """
+        Split the data into train, validation, and test sets.
+        """
+        # self._set_random_seed(self.random_seed)  # Set the random seed
+
+        # train_size = int(constants.TRAIN_DATASET * len(self.label_df))
+        # val_size = int(constants.VAL_DATASET * len(self.label_df))
+        # test_size = int(constants.TEST_DATASET * len(self.label_df))
+
+        # dataset = self.create_dataset()
+        # dataset = dataset.shuffle(buffer_size=len(self.label_df), seed=self.random_seed)
+
+        # train_dataset = dataset.take(train_size)
+        # val_dataset = dataset.skip(train_size).take(val_size)
+        # test_dataset = dataset.skip(train_size + val_size).take(test_size)
+
+        # return train_dataset, val_dataset, test_dataset
+
+        self._set_random_seed(self.random_seed)  # Set the random seed
+
+        unique_ids = self.label_df['study_id'].unique()
```

Comment:
Same here, this should be self.label_coordinates_csv. These two files are very different. See the count of lines in those files:

```
(vision) (vision) prashantkulkarni@vision-project-vm:/opt/dataset$ cat train.csv | wc -l
1976
(vision) (vision) prashantkulkarni@vision-project-vm:/opt/dataset$ cat train_label_coordinates.csv | wc -l
48693
```